lametro/models.py

```diff
@@ -185,65 +187,23 @@ def upcoming_board_meeting(cls):
                   .exclude(status='cancelled')\
                   .order_by('start_time').first()
 
-        # USED TO TEST THE CURRENT BOARD MEETING METHOD. Keep for now.
-        # faketime = datetime.now(app_timezone) - timedelta(days=30) - timedelta(hours=1)
-        # return cls.objects.filter(start_time__gt=faketime)\
-        #           .filter(name__icontains="Board Meeting")\
-        #           .order_by('start_time').first()
 
     @classmethod
     def current_meeting(cls):
-        # For testing...
-        # meeting_time = datetime.now(app_timezone) - timedelta(days=8) + timedelta(hours=2) + timedelta(minutes=13)
-        # meeting_end_time = meeting_time - timedelta(hours=3)
-        # print(meeting_time, "TIMEEEE")
-        
+        # A board meeting lasts 3 hours, and a committee event lasts 1 hour.
+        # Find all events occuring in the next three hours, and then filter accordingly. 
         meeting_time = datetime.now(app_timezone) + timedelta(minutes=6)
         meeting_end_time = datetime.now(app_timezone) - timedelta(hours=3)
```

Comment:
Your comment says find events occurring in the _next_ three hours.

`meeting_time` is six minutes from now. `meeting_end_time` is three hours ago.

So `found_events` is getting `Event` objects that start between three hours ago and six minutes from now, e.g., the last three hours, rather than the next three hours. Is this what we want?

Reply:
Yes, that's right. We can discuss this tomorrow - Metro had a lot of requests to get the logic like it is....it's also why the code ended up so verbose and needed refactoring.

Reply:
Sounds good to me!